relations/Cargo.toml

```diff
@@ -22,22 +22,24 @@ ark-serialize = { version = "^0.3.0", default-features = false }
 ark-snark = { version = "^0.3.0", default-features = false }
 ark-std = { version = "^0.3.0", default-features = false }
 blake2 = { version = "0.9.2", default-features = false }
+once_cell = "1.8"
+poseidon = { path = "../poseidon", default-features = false }
 
 [features]
 default = ["std"]
 std = [
-    "ark-bls12-381/std",
-    "ark-crypto-primitives/std",
-    "ark-ec/std",
-    "ark-ed-on-bls12-381/std",
-    "ark-ff/std",
-    "ark-gm17/std",
-    "ark-groth16/std",
-    "ark-marlin/std",
-    "ark-poly/std",
-    "ark-poly-commit/std",
-    "ark-r1cs-std/std",
-    "ark-relations/std",
-    "ark-serialize/std",
-    "blake2/std",
```

Comment:
do you have tab set to '  ' in toml formatting? this is kinda different to the whole codebase :P

Reply:
2 spaces for tab. Fixed it

Reply:
I hate tabs btw. wasteful